client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -133,113 +133,106 @@ export const Admin = () => {
 
   // Call when user asks to un-report a reported review. Accesses the Reviews database
   // and changes the reported flag for this review to false.
-  function unReportReview(review: Review) {
+  async function unReportReview(review: Review) {
+    const response = await axios.post('/api/admin/reviews/restore', {
+      review: review,
+      token: token,
+    })
 
-    //wz
-    axios
-      .post('/api/reviews/unreport', {
-        review: review,
-        token: token,
-      })
-      .then((response) => {
-        if (response.status === 200) {
-          const updatedReportedReviews = removeReviewFromList(
-            review,
-            reportedReviews
-          )
-          setReportedReviews(updatedReportedReviews)
-        }
-      })
+    if (response.status === 200) {
+      const updatedReportedReviews = removeReviewFromList(review, reportedReviews)
+      setReportedReviews(updatedReportedReviews)
+    }
   }
 
   // Call when user selects "Add New Semester" button. Runs code to check the
   // course API for new classes and updates classes existing in the database.
   // Should run once a semester, when new classes are added to the roster.
-  function addNewSem(semester: string) {
+  async function addNewSem(semester: string) {
```

Comment:
Just tested this endpoint and it seems like the `semester` parameter is inaccurate. I don't think we ever set this semester to a value with the `setAddSemester` variable.